pkg/apis/configuration/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2022 Kong, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the configuration.konghq.com v1alpha1 API group
+//+kubebuilder:object:generate=true
+//+groupName=configuration.konghq.com
```

Comment:
Generally we should be placing these directives somewhere else than between the package declaration and its godoc comments... are we doing this anywhere else? I suggest after the imports or the vars instead.

Reply:
This is how it's done in https://github.com/Kong/kubernetes-ingress-controller/blob/4a62fc748d70f4a356a2ff17a597e68ac9863829/pkg/apis/configuration/v1/groupversion_info.go#L18-L19 so I thought I'd make this consistent (not really thinking all that much about it TBH).

We can address that in a separate PR changing it in other files as well.

Reply:
Normally I'm OK with changing things in a follow up PR but this is so small and straightforward I would in fact ask that we do it here and in both places just to get it done and over with.

Reply:
Done, PTAL.